apps/ledger-live-desktop/tests/specs/speculos/swap.spec.ts

```diff
@@ -4,36 +4,94 @@ import { AppInfos } from "tests/enum/AppInfos";
 import { setExchangeDependencies } from "@ledgerhq/live-common/e2e/speculos";
 import { Fee } from "tests/enum/Fee";
 import { Swap } from "tests/models/Swap";
-import { Providers, Rates } from "tests/enum/Swap";
+import { Provider, Rates } from "tests/enum/Swap";
 
 const swaps = [
   {
-    swap: new Swap(Account.ETH_1, Account.BTC_NATIVE_SEGWIT_1, "0.015", Fee.MEDIUM),
+    swap: new Swap(
```

Comment:
Please attach corresponding test cases to every test in this file

Reply:
still waiting for those :sadge:
I'll create them this afternoon
cc @gbecerra-ledger @stual-ledger

Reply:
done just now